function-data-migration/src/main/java/com/future/function/data/migration/change/log/DataMigration_006.java

```diff
@@ -517,7 +517,7 @@ public void insertBatchAccessList(MongoTemplate mongoTemplate) {
              order = "0008")
   public void insertChatroomAccessList(MongoTemplate mongoTemplate) {
 
-    String urlRegex = "^\\/chatrooms(\\/)?$";
+    String urlRegex = "^(\\/m)?\\/chatrooms(\\/(\\w+)?)?$";
```

Comment:
Please move to separate migration file (should be `DataMigration_022` if not mistaken).

Reply:
so i need to separate between /chatrooms and /m/chatrooms ?
if yes why should i separate them if both have the same rule and can be written in one regex form ?

Reply:
Sorry, I think this may be a miscommunication.

My point is, as migration 006 has already been run, new migration class (022) must be created to register the updated url regex in this migration (006).